ReactSkia/views/common/RSkImageCacheManager.h

```diff
@@ -10,32 +10,41 @@
 #include "include/core/SkGraphics.h"
 #include "include/gpu/GrDirectContext.h"
 
+#include "ReactSkia/sdk/FollyTimer.h"
+
 #define SKIA_CPU_IMAGE_CACHE_LIMIT  50*1024*1024 // 52,428,800 bytes
 #define SKIA_GPU_IMAGE_CACHE_LIMIT  50*1024*1024 // 52,428,800 bytes
 
 using namespace std;
-
+using namespace rns::sdk;
 namespace facebook {
 namespace react {
 
-typedef facebook::better::map <string,sk_sp<SkImage>> ImageCacheMap;
+typedef struct imageDataExpiryTime {
```

Comment:
needs renaming ? relevant to the data stored in the structure 
decodedImageCacheData ?

Reply:
Done